build.sbt

```diff
@@ -251,7 +267,7 @@ lazy val commonScalacOptions = Seq(
   "-language:implicitConversions",
   "-language:experimental.macros",
   "-unchecked",
-  "-Xfatal-warnings",
+  //"-Xfatal-warnings",
```

Comment:
If you won't be fixing it in this PR, what about omitting this option from only the macros module?